### ISSUE TITLE
Exclude seems to not work

### ISSUE THREAD

**Issue by ciaoben** [davidolrik/sublime-rsync-ssh#13]
With these settings:
```javascript
{
	"folders":
	[
		{
			"follow_symlinks": true,
			"path": "/Applications/AMPPS/www/aaa/bbb"
		}
	],
	"settings":
	{
		"rsync_ssh":
		{
			"excludes":
			[
				"log/",
				"tmp/"
			],
			"options":
			[
			
			],
			"remotes":
			{
				"/Applications/AMPPS/www/aaa/bbb":
				[
					{
						"enabled": 1,
						"excludes":
						[ 
							
						],
						"options":
						[
						],
						"remote_host": "109.168.113.241",
						"remote_path": "/var/www/aaa/bbb",
						"remote_port": 22,
						"remote_post_command": "",
						"remote_pre_command": "",
						"remote_user": "ben"
					}
				]
			},
			"sync_on_save": true
		}
	}
}
```
I have this output. It seems to ignore my excludes!
```bash
rsync -v -zar -e ssh -q -p 22 -o ConnectTimeout=10 /Applications/AMPPS/www/aaa/bbb/ ben@109.168.113.241:/var/www/aaa/bbb
```

**Comment by davidolrik** [comment]
I've been trying to reproduce your problem, but to no avail. (I use excludes every day with no problem,)

I believe your excludes are there, but thet are not shown on the console.
I have fixed the debug output in cc758051b62ade86f78e4480fba9ec644addd7a1, please upgrade and let me know what the rsync command looks like in the console.

**Comment by davidolrik** [comment]
Since I haven't heard from you in 2 weeks, and I can't reproduce the problem, I'm going to assume that this is a non-issue.

If I'm wrong please let me know.
